### ISSUE TITLE
Snatched .torrent files drop in the root of /opt/sickrage instead of sending to torrent client

### ISSUE THREAD

**Issue by jbishop129** [SickRage/SickRage#3683]
Fresh install of SickRage via git, long time SR user here. Cannot figure out why when a show is Snatched, it does not go over to my configured uTorrent instance (configured, verified), but rather the .torrent file for the show is dropped in to the root of my SR install (/opt/sickrage). Log file shows the Torrent was successfully snatched. As we can see, it saves the result, rather than sending to uTorrent client:

```
2017-04-27 13:42:26 INFO::SEARCHQUEUE-BACKLOG-273181::Downloading Better.Call.Saul.S01E01.HDTV.x264-KILLERS[ettv] from ThePirateBay
2017-04-27 13:42:26 INFO::SEARCHQUEUE-BACKLOG-273181::Downloading a result from ThePirateBay at http://btdig.com/torrent/5B0555C7CD073E9C919259AE21B2A9AB51A62DE8.torrent
2017-04-27 13:42:27 INFO::SEARCHQUEUE-BACKLOG-273181::Downloading a result from ThePirateBay at http://itorrents.org/torrent/5B0555C7CD073E9C919259AE21B2A9AB51A62DE8.torrent
2017-04-27 13:42:28 INFO::SEARCHQUEUE-BACKLOG-273181::Saved result to Better.Call.Saul.S01E01.HDTV.x264-KILLERS[ettv].torrent
```

what am I missing? I know the error is on my end!

PS -- tried to post this in sickrage.ca/forums, but the registration isn't working. doesn't send verification email.

**Comment by sharkykh** [comment]
sickrage.ca is a **different** fork of SickRage.

You need to first check what fork you are using, do this by opening  http://localhost:8081/config/
On the bottom of the page there is "Website", verify that it's `http://sickrage.github.io/`

**Comment by sharkykh** [comment]
And on point, did you make sure on http://localhost:8081/config/search/#torrent-search, "Send torrents to" - uTorrent is actually selected?

The logs you posted don't contain any debug info. You need to enable debug logging in the config and try again, then post the results.

**Comment by jbishop129** [comment]
@sharkyh, thanks for replying. I will end up opening a new Issue, here's
where I'm at: My 'new' install of SR is in fact the other fork, so the
issue I am reporting, I am in the wrong place. I did check that it is set
to 'send torrents to.... uTorrent' and the uTorrent config successfully
'test connects' to my uTorrent server, so the behavior is odd, but
nonetheless, i'm in the wrong place.

Part two: My original SR install, which had been working flawlessly for
years, is in fact 'sickrage.github.io', and I believe at this time since
support over at the 'other fork' is spotty, I am best fit to fix my
original install (sickrage.github.io edition).
The issue I have with my original install, is out of the blue it is no
longer finding new episodes. I can manually search thepiratebay.org find my
episodes, send them to uTorrent (on the same VM as SR.github.io), and they
download, SR takes the ball from there and even handles the postprocessing
correctly too.
With that, the reason I stood up a new install was to see if it was
something with SR or something with my network and/or VPN. the new install
(albeit wrong fork... doh) snatches fine, but has the other issue (which is
fine, just needed to confirm the point of failure, I'm dumping the 'other
fork' install). I suspect logs are needed to track down the issue, any
chance you can point me in the right direction? I very much appreciate
everything the team does!

**Comment by sharkykh** [comment]
> I suspect logs are needed to track down the issue, any
chance you can point me in the right direction?

Yeah, sure:
1. First, make sure that SickRage (the correct fork) is up to date.
2. Enable debug logging in http://localhost:8081/config/general/#advanced-settings
3. Try searching - click on a magnifying glass on any episode.
4. Copy debug logs from here: http://localhost:8081/errorlogs/viewlog/?min_level=10
(You can click pause for so it stops updating the page, easier to copy)

**Comment by jbishop129** [comment]
@sharkykh Thanks again. This is now resolved: 
TL,DR; 
SSL error caused by old version of Ubuntu 14.04. 'do-release-upgrade' to 16.04 resolved the issue. 

Full version: I turned up debug logging and searched for an episode as you said, and among other things, the following error stood out:
`AA SSLError: [Errno 1] _ssl.c:510: error:14077438:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert internal error
...

2017-04-28 10:21:27 DEBUG SEARCHQUEUE-MANUAL-266189 :: [ThePirateBay] :: Request failed: [Errno 1] _ssl.c:510: error:14077438:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert internal error`

Google search lead me to [Issue #3647](https://github.com/SickRage/SickRage/issues/3647) where @prophead67 resolved a similar issue by doing the normal `sudo apt-get upgrade`. I started with that to no avail, and decided to go for broke with a full `do-release-upgrade`. Once the upgrade from 14.04 to 16.04 was complete and rebooted, I ran a search, which successfully Snatched the episode and queued it up for download. Note to self: Keep your systems up to date :)
